x-pack/plugin/src/test/resources/rest-api-spec/api/ilm.get_lifecycle.json

```diff
@@ -3,12 +3,12 @@
     "documentation": "http://www.elastic.co/guide/en/index_lifecycle/current/index_lifecycle.html",
     "methods": [ "GET" ],
     "url": {
-      "path": "/_ilm/{lifecycle}",
-      "paths": ["/_ilm/{lifecycle}", "/_ilm"],
+      "path": "/_ilm/{policy}",
```

Comment:
The filename here is still `get_lifecycle` - are we going to change those too?

Reply:
Personally I am still ok with put/get/delete lifecycle as the file names, I was only aiming to change the parameter. How strong are you feelings on this?

Reply:
Not very - otherwise I would have marked this as "requested changes".  I'm good with this as-is if you weren't intending to change it.